fastlane/lanes/ci_releases.rb

```diff
@@ -23,4 +23,15 @@
       environment: { RELEASE_VERSION: release_version }
     )
   end
+
+  lane :trigger_new_beta_in_ci do |release_version: release_version_current|
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: release_branch_name(release_version: release_version),
+      pipeline_file: File.join(PIPELINES_ROOT, 'new-beta-release.yml'),
+      message: "New beta for #{release_version}",
+      environment: { RELEASE_VERSION: release_version }
+    )
+  end
```

Comment:
So... this lane is not strictly necessary. After all, we use our internal release management app to trigger release builds.

I added it here because I plan to use it to test the new beta workflow once this PR lands as I don't want to reset the scenario for 4.53 in the release management app.

One could argue that it's useful to have these here as a fallback if the app is offline or something, but I think it might be overkill. In fact, maybe is best if I delete the whole file and use the Buildkite GUI to trigger this during testing.

What do you think?

WordPress has these lanes too, but that's because they were added before Rv2 was available. If we decide to remove the lanes from here, we shall delete them from there, too.

Reply:
Hmm I'm leaning toward deleting it.
I myself was confused for some time looking at the WPiOS automation, as we even mention it in the task in Rv2:
```
Run <code>bundle exec fastlane trigger_code_freeze_in_ci</code>.
```

While this lane is never really called, unless someone would do it directly (which [almost?] never happens these days).

Reply:
Done via 24fd4a6471928a84e70af3d1db58f3d8f725b484. See also https://github.com/wordpress-mobile/WordPress-iOS/pull/23589